src/components/filter/Filter.tsx

```diff
@@ -60,14 +60,17 @@ export const Filter = ({ filterTypes, initialFilters, setQueryVariables }: Props
   const [filters, setFilters] = useState(initialFilters);
   const [isCollapsed, setIsCollapsed] = useState(true);
 
-  useEffect(() => setQueryVariables(filters), [filters]);
+  useEffect(
+    () => setQueryVariables((prev) => ({ search: prev.search || '', ...filters })),
```

Comment:
wouldn't `() => setQueryVariables((prev) => ({ ...prev, ...filters })),` be fine?